app/models/services/locators/doi_paper_locator.rb

```diff
@@ -6,30 +6,79 @@ def column
   def find_paper
     return super if super
 
+    paper = Paper.new
+
+    crossref = Crossref.new
+    if (result = crossref.get_doi_metadata(self.locator_id))
+      data = result['message']
+      paper = self.import_data_to_paper(paper,data,'crossref')
+    end
+
     pubmed = Pubmed.new
-    if uid = pubmed.find_uid_by_doi(self.locator_id)
+    if (uid = pubmed.find_uid_by_doi(self.locator_id))
```

Comment:
It does, but let's use the proper level of error handling. https://github.com/briankung/projectcredo/pull/99/files/421f3270169e3dadbce1f99dff2edfbee31b426c#r82536967